state/state_test.go

```diff
@@ -2180,7 +2192,8 @@ func TestExecutorGasRefund(t *testing.T) {
 		EthTimestamp:     uint64(time.Now().Unix()),
 		UpdateMerkleTree: 1,
 		ChainId:          stateCfg.ChainID,
-		ForkId: 		 forkID,
+		ForkId:           forkID,
```

Comment:
I don't know, maybe it would be better to create an issue to review commented tests in the state and try to find in the history why they are commented at the moment. I just added the fields to the commented tests to help us in case we want to uncomment them in the future.